src/azul/plugins/metadata/anvil/__init__.py

```diff
@@ -526,7 +555,10 @@ class AnvilFile(File):
     @classmethod
     def from_hit(cls, hit: JSON) -> Self:
         return cls(uuid=hit['document_id'],
-                   version=hit['version'],
+                   # If this method is called after MetadataPlugin.modify_hit()
```

Comment:
I think it is a bad idea to alter this method to make it compatible with the test.